spec/spec_helper.rb

```diff
@@ -4,11 +4,14 @@
 ENV['environment'] ||= 'test'
 
 require 'bundler/setup'
+require 'active_fedora/cleaner'
 require 'darlingtonia'
 
 Dir['./spec/support/**/*.rb'].each { |f| require f }
 
 RSpec.configure do |config|
   config.filter_run focus: true
   config.run_all_when_everything_filtered = true
+
```

Comment:
I think we need to recreate a default admin set as part of our cleaning process, works can't be created without one. I think it might be something we should add to our general test process. I'm not sure how much that matters here, but it might.

Reply:
At this point, none of this code knows about anything in Hyrax other than `Hyrax::Configuration` (which is optional). There will probably be a point where we need to handle a default `AdminSet` for test setup, but at this point its blissfully unnecessary.